### PR TITLE
Redirect all server urls to https. Fix certbot cron job

### PR DESCRIPTION
**Pull request by boatrite**
Cron job no longer is --quiet and gets rid of quotes around command
that I think are causing it to fail

### CODE REVIEW

**Comment by petitJAM** [comment]
:ship: ? idk if you want anyone else looking at it

**Comment by boatrite** [comment]
Probably should have @samsonasu check it out too. Stuff going into subspace is pretty important; I like having his eyes on it.

File: ansible/roles/letsencrypt/tasks/main.yml [hunk 1]
```diff
@@ -78,24 +78,16 @@
...
       insertbefore: "</VirtualHost>"
     sudo: true
+    with_items: "{{ ([server_name] + server_aliases) }}"
     notify: apache restart
 
   - name: Setup cron job to auto renew
     cron:
       name: Auto-renew SSL
-      job: sudo "{{certbot_dir}}/certbot-auto renew --quiet --no-self-upgrade"
+      job: sudo {{certbot_dir}}/certbot-auto renew --no-self-upgrade
```
Comment (samsonasu) [inline ansible/roles/letsencrypt/tasks/main.yml]: This should be installed on root so it doesn't need the sudo

File: ansible/roles/letsencrypt/tasks/main.yml [hunk 2]
```diff
@@ -78,24 +78,16 @@
...
-
-  - name: Force redirect to https (3/3)
-    lineinfile:
-      dest: /etc/apache2/sites-available/{{project_name}}.conf
-      line: "RewriteRule ^ https://%{SERVER_NAME}%{REQUEST_URI} [END,QSA,NE,R=permanent]"
+      line: "RewriteCond %{SERVER_NAME} ={{item}}\nRewriteRule ^ https://%{SERVER_NAME}%{REQUEST_URI} [END,QSA,NE,R=permanent]"
       state: present
       insertbefore: "</VirtualHost>"
     sudo: true
+    with_items: "{{ ([server_name] + server_aliases) }}"
```
Comment (samsonasu) [inline ansible/roles/letsencrypt/tasks/main.yml]: I don't know what this does and I think it's probably not correct. did you test it?

Comment (boatrite) [inline ansible/roles/letsencrypt/tasks/main.yml]: Yeah I tested it and it works. `server_aliases` is an array, `server_name` is not, so I make `server_name` an array then add on the aliases. We also do this a few lines above in the certbot command to set the domains.

File: ansible/roles/letsencrypt/tasks/main.yml (hunk 1 above)
Comment (boatrite) [inline ansible/roles/letsencrypt/tasks/main.yml]: :+1:

**Pull request merged**